climada/engine/unsequa/calc_base.py

```diff
@@ -423,6 +425,16 @@ def sensitivity(
 
         return sens_output
 
+    @abstractmethod
+    def uncertainty(self):
+        """Compute the uncertainty of the output values"""
+        pass
+
+    @abstractmethod
```

Comment:
private _and_ abstract seems somehow contradicting. if we go for it, `compute_metrics` ought to be public.